requirements/collections_requirements.yml

```diff
@@ -16,3 +16,7 @@ collections:
     version: 0.4.0  # first to contain necessary grouping and filtering features
   - name: ovirt.ovirt
     version: 1.0.0  # contains naming fix, was originally named ovirt.ovirt_collection
+  - name: community.kubernetes
+    version: 0.11.0
+  - name: ansible.posix
+    version: 0.1.0
```

Comment:
let's comment that these are required for the isolated management playbooks.

I think it makes sense to combine them right now - but they will be separated back out as execution environments start to be used for inventory imports.

Reply:
left comments